deeprank/features/ResidueContacts.py

```diff
@@ -0,0 +1,384 @@
+import logging
+import pdb2sql
+import re
+import os
+
+import numpy
+
+from deeprank.features.FeatureClass import FeatureClass
+from deeprank.parse.param import ParamParser
+from deeprank.parse.top import TopParser
+from deeprank.parse.patch import PatchParser
+from deeprank.models.patch import PatchActionType
+
+_log = logging.getLogger(__name__)
+
+
+class ResidueSynonymCriteria:
+    def __init__(self, residue_name, atoms_present, atoms_absent):
+        self.residue_name = residue_name
```

Comment:
Could you add docstrings on the class and methods ? We use them to generate the doc (I know it's a boring task but it's rather useful if someone else work on these files later on). I don't remember which flavor of docstring we use but if you could check on the other files and use the same that would be great !

Reply:
Yes! I will..